docs_2.0/manual_EN/3.quick-start/2.deploy-nebula-graph-with-docker-compose.md

```diff
@@ -181,4 +182,4 @@ docker pull vesoft/nebula-console:v2-preview-nightly
 
 ### Q2: How to update the docker images of Nebula Graph services
 
-To update the images of graphd, storaged, and metad, run `docker-compose pull` in the `nebula-docker-compose` directory.
+To update the images of the Graph Service, Storage Service, and Meta Service, run `docker-compose pull` in the `nebula-docker-compose` directory.
```

Comment:
The lower case would do just fine. No need to capitalize them.

Reply:
We add this add our terms.
@oldLady344 @whitewum

Reply:
let's vote.